src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -592,6 +592,11 @@ public uint MaxPageSize()
         return (uint?)Runtime?.Pagination?.MaxPageSize ?? PaginationOptions.MAX_PAGE_SIZE;
     }
 
+    public bool NextLinkRelative()
```

Comment:
Please fix PR title to reflect the support added to nextLink being relative using `next-link-relative` property